migrations/000006.php

```diff
@@ -0,0 +1,11 @@
+<?php
+// This migration adds a "activated" column to the subscriptions table and sets all values to true.
+
+/** @noinspection PhpUndefinedVariableInspection */
+$columnQuery = $db->query("SELECT * FROM pragma_table_info('subscriptions') WHERE name='inactive'");
+$columnRequired = $columnQuery->fetchArray(SQLITE3_ASSOC) === false;
+
+if ($columnRequired) {
+    $db->exec('ALTER TABLE subscriptions ADD COLUMN inactive BOOLEAN DEFAULT false');
+    $db->exec('UPDATE subscriptions SET inactive = false');
+}

```

Comment:
Yes! That's it. Thank you so much.
But meanwhile we have 2 new migrations so `000006.php` is conflicting with another migration.
Can you please rename the file to `000008.php` ?

Thank you.

Reply:
renamed